contrib/rpm/podman.spec

```diff
@@ -0,0 +1,207 @@
+export GOPATH=$(pwd)/_build:$(pwd):$(pwd):%{gopath}
+export BUILDTAGS="selinux seccomp $(hack/btrfs_installed_tag.sh) $(hack/btrfs_tag.sh) $(hack/libdm_tag.sh) containers_image_ostree_stub"
+
+GOPATH=$GOPATH BUILDTAGS=$BUILDTAGS %gobuild -o bin/%{name} %{import_path}/cmd/%{name}
+
+BUILDTAGS=$BUILDTAGS make all
+
+%install
+PREFIX=%{buildroot}/usr
+%make_install PREFIX=${PREFIX}
+%make_install PREFIX=${PREFIX} install.completions
+%make_install PREFIX=${PREFIX} install.cni
+%make_install PREFIX=${PREFIX} install.man
+# We don't want to ship conmon
+rm -f %{buildroot}/usr/libexec/crio/conmon
+rm -rf %{buildroot}/usr/lib/debug/usr/libexec/crio
+
+# source codes for building projects
+%if 0%{?with_devel}
+install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
+
+echo "%%dir %%{gopath}/src/%%{import_path}/." >> devel.file-list
+# find all *.go but no *_test.go files and generate devel.file-list
+for file in $(find . \( -iname "*.go" -or -iname "*.s" \) \! -iname "*_test.go" | grep -v "vendor") ; do
+    dirprefix=$(dirname $file)
+    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$dirprefix
+    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
+    echo "%%{gopath}/src/%%{import_path}/$file" >> devel.file-list
+
+    while [ "$dirprefix" != "." ]; do
+	echo "%%dir %%{gopath}/src/%%{import_path}/$dirprefix" >> devel.file-list
+	dirprefix=$(dirname $dirprefix)
+    done
+done
+%endif
+
+# testing files for this project
+%if 0%{?with_unit_test} && 0%{?with_devel}
+install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
+# find all *_test.go files and generate unit-test-devel.file-list
+for file in $(find . -iname "*_test.go" | grep -v "vendor") ; do
+    dirprefix=$(dirname $file)
+    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$dirprefix
+    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
+    echo "%%{gopath}/src/%%{import_path}/$file" >> unit-test-devel.file-list
+
+    while [ "$dirprefix" != "." ]; do
+	echo "%%dir %%{gopath}/src/%%{import_path}/$dirprefix" >> devel.file-list
+	dirprefix=$(dirname $dirprefix)
+    done
+done
+%endif
+
+%if 0%{?with_devel}
+sort -u -o devel.file-list devel.file-list
+%endif
+
+%check
+%if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
+export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+%endif
+
+%if ! 0%{?gotest:1}
+%global gotest go test
+%endif
+
+%gotest %{import_path}/cmd/podman
+%gotest %{import_path}/libpod
+%gotest %{import_path}/pkg/registrar
+
+#define license tag if not already defined
+%{!?_licensedir:%global license %doc}
+
+%files
+%license LICENSE
+%doc README.md CONTRIBUTING.md hooks.md install.md code-of-conduct.md transfer.md
+%{_bindir}/podman
+%{_mandir}/man1/*.1*
+%{_datadir}/bash-completion/completions/*
+%{_sysconfdir}/cni/net.d/87-podman-bridge.conflist
+
+%if 0%{?with_devel}
+%files -n libpod-devel -f devel.file-list
+%license LICENSE
+%doc README.md CONTRIBUTING.md hooks.md install.md code-of-conduct.md transfer.md
```

Comment:
Should we include commands.md?

Reply:
Not sure we should provide any of these.
Removing them all.